### DISCUSSION TITLE
Disable search indexing on versioned documentation

### DISCUSSION THREAD

**Discussion by LoicPoullain** [facebook/docusaurus#7960] [Q&A] [answered]
Hi everyone 👋 

Currently, the pages of the versioned documentation of [FoalTS](https://foalts.org) are still indexed on Google. So, when someone does a search on Google, in some cases, it comes across the documentation of deprecated versions.

Is it it possible to disable search indexing of versioned documentation?

The only thing that I see at the moment to do this is to add a `meta` tag on _each_ page: https://docusaurus.io/docs/seo#single-page-metadata. Is there an easier solution?

**Comment by slorber** [comment, marked as answer]
That makes sense, will add it now the ability to use `noIndex: true` on version config

Are your deprecated versions docs totally different from the new version docs? Do your older links get backlinks contributing to SEO? In these cases it might be better to point older pages to newer pages with a custom canonical URL 🤷‍♂️  Unfortunately Google does not make it very clear how to handle versioned documentation

Might be a little related to "version aliases" that has been discussed in the past here: https://github.com/facebook/docusaurus/issues/3285

**Reply by slorber** [reply]
fixed in https://github.com/facebook/docusaurus/pull/7963: you can use the docs plugin options to noindex a specific version

**Reply by LoicPoullain** [reply]
Thank you very much for your quick reply @slorber and for implementing the feature so quickly!

**Reply by LoicPoullain** [reply]
> Are your deprecated versions docs totally different from the new version docs?

It depends. Some pages exist in both current and older versions. Other do not exist anymore or have been replaced with pages with other names.
